pkg/flexvolume/client.go

```diff
@@ -22,43 +39,119 @@ func (c *client) Init() error {
 }
 
 func (c *client) Attach(jsonOptions map[string]string) error {
+	var output []byte
 	_, err := c.apiClient.Attach(
 		context.Background(),
 		&AttachRequest{
 			JsonOptions: jsonOptions,
 		},
 	)
+	if err == nil {
+		a := AttachSuccessOutput{
+			Status: "Success",
+			Device: jsonOptions["volumeID"],
```

Comment:
If we have an invalid volumeID, then the flexvolume server itself will catch it and return an error. We reach this if condition only when everything went good and we actually had a volumeID as a key.